cmd/commands/use.go

```diff
@@ -87,27 +86,6 @@ func useCmd(ctx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("%s not supported, error: %w", name, err)
 	}
-	if version == "" {
-		list := source.List()
-		var arr []string
-		for _, version := range list {
-			arr = append(arr, string(version))
-		}
-		selectPrinter := pterm.InteractiveSelectPrinter{
-			TextStyle:     &pterm.ThemeDefault.DefaultText,
-			OptionStyle:   &pterm.ThemeDefault.DefaultText,
-			Options:       arr,
-			DefaultOption: "",
-			MaxHeight:     5,
-			Selector:      "->",
```

Comment:
This piece of code should not be moved to `sdk.use`, because it exists as an additional function of the command. Because there may be GUI projects later, if you do this, it will not be easy to expand.

Reply:
but if we want plugin can read `.nvmrc` etc file, the PreUse hooks should be executed before the select ui shown, so we should invoke PreUse explicitly in use command